app/src/main/kotlin/at/bitfire/davdroid/ui/webdav/WebdavMountsActivity.kt

```diff
@@ -122,81 +117,238 @@ class WebdavMountsActivity: AppCompatActivity() {
+        var showingDialog by remember { mutableStateOf(false) }
+        if (showingDialog) {
+            AlertDialog(
+                onDismissRequest = { showingDialog = false },
+                title = { Text(stringResource(R.string.webdav_remove_mount_title)) },
+                text = { Text(stringResource(R.string.webdav_remove_mount_text)) },
+                confirmButton = {
+                    TextButton(
+                        onClick = {
+                            Logger.log.log(Level.INFO, "User removes mount point", info.mount)
+                            model.remove(info.mount)
+                        }
+                    ) {
+                        Text(stringResource(R.string.dialog_remove))
+                    }
+                },
+                dismissButton = {
+                    TextButton(
+                        onClick = { showingDialog = false }
+                    ) {
+                        Text(stringResource(R.string.dialog_deny))
+                    }
+                }
+            )
+        }
 
-                binding.quota.visibility = View.VISIBLE
-                binding.quota.text = context.getString(R.string.webdav_mounts_quota_used_available,
-                    FileUtils.byteCountToDisplaySize(quotaUsed),
-                    FileUtils.byteCountToDisplaySize(quotaAvailable)
+        Card(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(8.dp),
+            backgroundColor = MaterialTheme.colors.onSecondary
+        ) {
+            Column(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(16.dp)
+            ) {
+                Text(
+                    text = info.mount.name,
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(bottom = 8.dp),
+                    style = MaterialTheme.typography.body1
                 )
-            } else {
-                binding.quotaProgress.visibility = View.GONE
-                binding.quota.visibility = View.GONE
-            }
-
-            binding.browse.setOnClickListener {
-                val intent = Intent(Intent.ACTION_OPEN_DOCUMENT)
-                intent.addCategory(Intent.CATEGORY_OPENABLE)
-                intent.type = "*/*"
-                val uri = DocumentsContract.buildRootUri(authority, info.mount.id.toString())
-                intent.putExtra(DocumentsContract.EXTRA_INITIAL_URI, uri)
-                model.browseIntent.value = intent
-                model.browseIntent.value = null
-            }
+                Text(
+                    text = info.mount.url.toString(),
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(bottom = 8.dp),
+                    style = MaterialTheme.typography.caption,
+                    overflow = TextOverflow.Ellipsis,
+                    maxLines = 1
```

Comment:
Hm the previous ellipse was "middle", which is important (because you can't see the important last URL segment otherwise). Seems that this is not supported by Compose. I'd leave it away completely, let's show the full URL.